tests/end-to-end/harbor/use-ui.cy.js

```diff
@@ -19,8 +19,10 @@ describe("harbor ui", function() {
 
         cy.contains("log out", opt)
           .click()
-
-        cy.harborAdminLogin(this.ingress)
+          .then(() => {
+            cy.contains("LOGIN VIA LOCAL DB", opt)
+            cy.harborAdminLogin(this.ingress)
+          })
```

Comment:
Is this needed for some reason?
If it is just to wait for the logout to finalise you should be able to have the `cy.contains(...)` after the click right?